lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx

```diff
@@ -26,6 +26,11 @@ describe('NxCollapsibleItems', function() {
     expect(NxCollapsibleItems).toBe(NxTreeView);
   });
 
+  it('has NxCollapsibleItemsChild that is aliased as NxTreeViewChild', function() {
+    expect(NxCollapsibleItemsChild).toBe(NxTreeViewChild);
```

Comment:
`NxCollapsibleItemsChild` isn't publicly exported, so it doesn't need to be tested and shouldn't be the one mentioned in the test name

Reply:
> `NxCollapsibleItemsChild` isn't publicly exported, so it doesn't need to be tested and shouldn't be the one mentioned in the test name

Oh it is publicly exported. We added the Child sub-component on the rename NxTreeView ticket and we still export the individual component so it's not a breaking change (I think).

Reply:
> Oh it is publicly exported. We added the Child sub-component on the rename NxTreeView ticket and we still export the individual component so it's not a breaking change (I think).

`NxTreeViewChild` is still exported but `NxCollapsibleItemChild` isn't.  It is only externally available as `NxCollapsibleItems.Child`